app/assets/javascripts/modules/image_x_viewer.js

```diff
@@ -21,6 +21,13 @@
       });
     };
 
+    var _setupButtonListeners = function() {
+      $el.parent().parent().find('[data-sul-view-mode]').on('click', function(e, f) {
```

Comment:
Line is too long.<br>'f' is defined but never used.<br>'e' is defined but never used.